lib/nexmo_developer/app/views/layouts/partials/_locale_switcher.html.erb

```diff
@@ -11,7 +11,7 @@
       </span>
     <% end %>
   </div>
-  <% unless presenter.disabled? %>
+  <% unless presenter.disabled? || presenter.multiple_locales? == false %>
```

Comment:
@bencgreenberg I think the idea is to hide the selector completely instead of disabling it.

Reply:
👍